src/components/DefaultModal.tsx

```diff
@@ -3,11 +3,15 @@ import styled, { ThemeContext } from 'styled-components';
 import MainButton, { Variant } from './MainActionButton';
 
 export const Overlay = styled.div`
-	position: absolute;
+	position: fixed;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
 	backdrop-filter: blur(5px);
 	background-color: rgba(10, 10, 20, 0.5);
 	backdrop-filter: blur(0.2em);
-	height: 100%;
+	height: 100vh;
```

Comment:
Should this be 100%? What happens if we remove this altogether and rely on the position: fixed and top,left,right,bottom properties?

Reply:
It still works! Just removed the height now :)